scripts/zones/Pashhow_Marshlands/mobs/Brass_Quadav.lua

```diff
@@ -3,9 +3,30 @@
 --  Mob: Brass Quadav
 -----------------------------------
 require("scripts/globals/regimes")
+require("scripts/globals/follow")
+local ID = require("scripts/zones/Pashhow_Marshlands/IDs")
 -----------------------------------
 local entity = {}
 
+entity.onMobRoamAction = function(mob)
+    if mob:getID() == ID.mob.BOWHO_WARMONGER + 1 then
```

Comment:
actually, I think I had him do offsets, since they always follow up in IDs list and won't shift. I'd have to expand my own prior comments but seems familiar and like something I'd say

Reply:
If so then this can be ignored :)